articles/container-apps/tutorial-scaling.md

```diff
@@ -291,6 +295,156 @@ The following screenshot shows a zoomed view of how the requests received by you
 
 :::image type="content" source="media/scale-app/azure-container-apps-scale-replicas-metrics-3.png" alt-text="Screenshot of container app metrics graph, showing requests split by replica, in a zoomed view.":::
 
+## CPU and memory scaling
+
+You should prefer [HTTP scale rules](/azure/container-apps/scale-app#http) to CPU or memory scale rules when possible.
+
+When you use the Azure CLI to add a scale rule to a container app that already has a scale rule, the new scale rule replaces the old scale rule. To see how to add multiple scale rules, see [Multiple scale rules](#multiple-scale-rules).
+
+After you add a CPU or memory scale rule, you can test it by [sending requests to your container app](#send-requests) and [viewing the scaling in Azure portal](#view-scaling-in-azure-portal-optional).
```

Comment:
Since it's the quickstart container, I doubt sending it requests will make it consume more or less CPU/memory.